packages/dashboard/src/app/views/editorSettings/archive/test/archive.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { fireEvent, screen } from '@testing-library/react';
+import { FlagsProvider } from 'flagged';
+
+/**
+ * Internal dependencies
+ */
+import { renderWithProviders } from '../../../../../testUtils';
+import Archive from '..';
+
+describe('Editor Settings: <Archive />', function () {
+  it('should not render anything if experiment is not enabled', function () {
+    renderWithProviders(<Archive updateSettings={jest.fn()} isEnabled />);
```

Comment:
nit: could just do 
`const mockUpdateSettings = jest.fn();` 
and then use that in all of these `updateSettings`

Reply:
This is a copy and paste from another existing page